apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.IntegrationTests/ErrorReporting/ErrorEventEntryVerifiers.cs

```diff
@@ -14,13 +14,36 @@
 
 using Google.Cloud.ClientTesting;
 using Google.Cloud.ErrorReporting.V1Beta1;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http;
 using Xunit;
 
 namespace Google.Cloud.Diagnostics.Common.IntegrationTests
 {
     public static class ErrorEventEntryVerifiers
     {
+        /// <summary>
+        /// Checks that there's only one event present.
+        /// Includes a message with context info to help track the possible issue.
+        /// </summary>
+        public static ErrorEvent VerifySingle(IEnumerable<ErrorEvent> events, string contextInfo)
+        {
+            var errorEvent = events.SingleOrDefault();
+            Assert.True(errorEvent != null, $"No entries found for context {contextInfo}");
+            return errorEvent;
+        }
+
+        /// <summary>
+        /// Checks that there are <paramref name="expected"/> events present.
+        /// Includes a message with context info to help track the possible issue.
+        /// </summary>
+        public static IEnumerable<ErrorEvent> VerifyMany(IEnumerable<ErrorEvent> events, int expected, string contextInfo)
```

Comment:
It might be worth converting the `IEnumerable<ErrorEvent>` to ` List<ErrorEvent>` using `ToList()`, so that it's only evaluated once.

Reply:
OK